src/Microsoft.Azure.WebJobs.Extensions.Rpc/WebJobs.Extensions.Rpc.csproj

```diff
@@ -5,9 +5,9 @@
     <TargetFramework>net6.0</TargetFramework>
     <LangVersion>latest</LangVersion>
     <PackageId>Microsoft.Azure.WebJobs.Extensions.Rpc</PackageId>
+    <Description>This package provides RPC capabilities to the WebJobs SDK, allowing extensions to communicate between the host and worker via RPC. For more information, please visit https://learn.microsoft.com/en-us/azure/app-service/webjobs-sdk-get-started.</Description>
     <AssemblyName>Microsoft.Azure.WebJobs.Extensions.Rpc</AssemblyName>
     <RootNamespace>Microsoft.Azure.WebJobs.Extensions.Rpc</RootNamespace>
-    <Description>This package contains the runtime assemblies for Microsoft.Azure.WebJobs.Extensions.Rpc. For more information, please visit http://go.microsoft.com/fwlink/?LinkID=320971</Description>
```

Comment:
As part of updating the Descriptions, I'm also updating the "more information" URI to point to our official WebJobs SDK tutorial. Existing description points to wiki.